utils.go

```diff
@@ -27,3 +29,13 @@ func waitForAliveVMM(ctx context.Context, client *Client) error {
 		}
 	}
 }
+
+// envValueOrDefaultTime check if env value exists and returns it or returns default value
+// provided as a second param to this function
+func envValueOrDefaultTime(envName string, def int) time.Duration {
```

Comment:
I wonder if it makes sense to do this during `NewMachine` and set that value somewhere on the machine so we don't have to do this multiple times. That way defaults are all happening in the same place. What do you think?

Reply:
Hmm, I can't say that I like that idea, to have a global state, especially few of them like in this case, it's not so expensive to make this call on each request.

Reply:
I think it is less about it being more efficient, but a preference of having defaults happening in the same place, `NewClient`. That way in the future we can change the default in a single place. Currently if we rename the function, `envValueOrDefaultInt`, to something else, we'd have to change it in several places. So, I think I prefer that it was done in `NewClient` and have a private field that houses the values.